sycl/source/detail/pi.cpp

```diff
@@ -451,7 +451,8 @@ static void initializePlugins(std::vector<plugin> &Plugins) {
     if (trace(TraceLevel::PI_TRACE_BASIC))
       std::cerr << "SYCL_PI_TRACE[basic]: "
                 << "Plugin found and successfully loaded: "
-                << PluginNames[I].first << std::endl;
+                << PluginNames[I].first << " ["
+                << Plugins[I].getPiPlugin().PluginVersion << "]" << std::endl;
```

Comment:
I suggest being a bit more verbose and say that the printed value is the version of the plugin.
Also, I think we want to "standardize" what the plugin version is, so that all plugins follow it.
So, maybe the concatenating macro should be moved to pi.h out of pi_cuda.hpp?